content/swift/concepts/sets/terms/subtracting/subtracting.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: '.subtracting()'
+Description: 'Returns a new set containing the elements from the target set that are not in the given set.'
+  - 'Mobile Development'
+  - 'Computer Science'
```

Comment:
```suggestion
  - 'Computer Science'
Subjects:
  - 'Mobile Development'
```

We were missing a list of subjects here. Adding this should get the "test" GitHub check to pass. 😄